src/config.json

```diff
@@ -1,21 +1,18 @@
 {
-  "api": {
-    "host": "default"
-  },
-  "resources": {
-    "host_thumbnails": "http://localhost:4567/thumbnails/:s",
-    "host_objects": "http://localhost:4567/objects/:o"
-  },
   "competition": {
-    "host": "",
-    "tls": true,
-    "log": false,
+    "host": "localhost:8080",
```

Comment:
This flag should not be set in the default config

Reply:
Right, that slipped through. Has been removed